hathor/manager.py

```diff
@@ -140,7 +140,7 @@ def __init__(self,
         self.reactor = reactor
         add_system_event_trigger = getattr(self.reactor, 'addSystemEventTrigger', None)
         if add_system_event_trigger is not None:
-            add_system_event_trigger('after', 'shutdown', self.stop)
+            add_system_event_trigger('after', 'shutdown', self.try_stop)
```

Comment:
This issue is a little bit more complex, I can split it into a different PR to give it more context, it doesn't affect this feature anymore (since I've disabled the original signal hooks which prevented SIGINT from working). On the different PR I can give it more context.

Reply:
I've removed this set of changes (try_stop and sigquit handler). I'll open a separate PR.

Reply:
Ok! Thanks!